api/desecapi/models.py

```diff
@@ -373,18 +374,23 @@ class Meta:
 
 # RR set types: the good, the bad, and the ugly
 # known, but unsupported types
-RR_SET_TYPES_UNSUPPORTED = {'ALIAS', 'DNAME'}
+RR_SET_TYPES_UNSUPPORTED = {
+    'ALIAS',  # Requires signing at the frontend, hence unsupported in desec-stack
+    'DNAME',  # "no not combine with DNSSEC", https://doc.powerdns.com/authoritative/settings.html#dname-processing
+    'IPSECKEY',  # broken in pdns, https://github.com/PowerDNS/pdns/issues/9055
+    'KEY',  # Application use restricted by RFC 3445, DNSSEC use replaced by DNSKEY and handled automatically
+}
 # restricted types are managed in use by the API, and cannot directly be modified by the API client
 RR_SET_TYPES_AUTOMATIC = {
     # automatically managed by the backend:
-    'CDNSKEY', 'CDS', 'DNSKEY', 'NSEC', 'NSEC3', 'OPT', 'RRSIG',
+    'CDNSKEY', 'CDS', 'DNSKEY', 'KEY', 'NSEC', 'NSEC3', 'OPT', 'RRSIG',
     # automatically managed by the API:
     'NSEC3PARAM', 'SOA'
 }
 # backend types are types that are the types supported by the backend(s)
 RR_SET_TYPES_BACKEND = pdns.SUPPORTED_RRSET_TYPES
 # validation types are types supported by the validation backend, currently: dnspython
-RR_SET_TYPES_VALIDATION = set(rdatatype._by_text.keys())
+RR_SET_TYPES_VALIDATION = set(ANY.__all__) | set(IN.__all__)
```

Comment:
for clarity, better import `rdtypes` and write `rdtypes.ANY` and `rdtypes.IN` here

Reply:
yeah I wish!! But:

```python3
>>> from dns import rdtypes
>>> rdtypes.ANY
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'dns.rdtypes' has no attribute 'ANY'
```

No idea what's going on here. But I decided it's not so important. If you can clarify my confusion here, that would be highly appreciated :)

Reply:
I guess ANY is just not a member of `rdtypes`, but rather a submodule (that does not import ANY itself)